### ISSUE TITLE
Statement does not exist in ABAPv755(or a parser error), "RAISE" (parser_error)

### ISSUE THREAD

**Issue by gregorwolf** [abaplint/abaplint#2155]
In this code which is part of [abapGitTestSEGWservice](https://github.com/gregorwolf/abapGitTestSEGWservice):

```ABAP
  METHOD test.
    TRY.
        DATA(lr_reader) = NEW zcl_excel_reader_2007( ).
        DATA(lr_excel) = reader->load( i_excel2007 = i_xlsx ).
        DATA(lr_worksheet) = excel->get_worksheet_by_name( ip_sheet_name = c_metadata_sheetname ).
        IF lr_worksheet IS INITIAL.
          RAISE EXCEPTION NEW zcx_excel( i_message = 'Worksheet not found.').
        ENDIF.
      CATCH zcx_excel INTO DATA(lr_ex).
        RAISE EXCEPTION NEW zexcel_du_ex( i_message = lr_ex->get_text( ) ).
    ENDTRY.
  ENDMETHOD.
```

the RAISE EXCEPTION statement in the IF statement causes:

`Statement does not exist in ABAPv755(or a parser error), "RAISE" (parser_error)`

The RAISE EXCEPTION in the CATCH doesn't cause an error.

**Comment by larshp** [comment]
quick workaround, add a space, change
`RAISE EXCEPTION NEW zcx_excel( i_message = 'Worksheet not found.').`
to
`RAISE EXCEPTION NEW zcx_excel( i_message = 'Worksheet not found.' ).`

keep the issue open, its a bug in tabaplint

**Comment by larshp** [comment]
the fix will be allowing the no-space scenario in the parser

and then rule https://rules.abaplint.org/parser_missing_space/ will report an error

**Comment by larshp** [comment]
@gregorwolf try again, 2.79.4

**Comment by larshp** [comment]
@gregorwolf
